Remotion/Web/Development.WebTesting/Configuration/WebTestConfigurationSection.cs

```diff
@@ -36,6 +36,7 @@ public class WebTestConfigurationSection : ConfigurationSection
     private readonly ConfigurationProperty _commandTimeoutProperty;
     private readonly ConfigurationProperty _downloadStartedTimeoutProperty;
     private readonly ConfigurationProperty _downloadUpdatedTimeoutProperty;
+    private readonly ConfigurationProperty _verifyWebApplicationStartedTimeoutProperty;

```

Comment:
This might be obvious, but why are there no changes to configuration files?

Reply:
Because i initally thought, the default is ok.
But thinking again about it, we should probably take a reasonable, slightly higher default value and some lower values for re-motion in the config files.